docs/source/_src_user_guide/state_propagation/running_simulation/propagation_architecture.rst

```diff
@@ -18,7 +18,7 @@ Before starting the propagation, several steps are taken to initialize the varia
   
   * An ``EnvironmentUpdater`` object (in C++; not exposed to Python), which decided which environment models need to be updated for each function evaluation (:ref:`single_propagation_evaluation:` and how)
```

Comment:
single_propagation_evaluation reference not working

Reply:
Fixed (see above)